### ISSUE TITLE
Data overlap among train, dev and test?

### ISSUE THREAD

**Issue by hdzhang-code** [yinwenpeng/BenchmarkingZeroShot#5]
Thanks for your work, I have learned a lot from it.

However, there is something werid in the dataset splitting. For situation, why is there huge amount of duplicate data in test+dev and train. For example,

The following data shows up in both
/situation/train_pu_half_v0.txt and /situation/test.txt

"water vegetation normally found around small watering holes is scarce because much of the water is depleted or has become too salty ."

In the paper, Table 5, the performance of Binary-BERT is reported on test set, after it's trained on train set. If there is huge amount of duplicate data in test set and training set, how does the reported performance make sense?

**Comment by yinwenpeng** [comment]
Hi, this "situation" task is a multi-label task. A piece of text can have multiple labels. Zero-shot or few-shot are often for labels (which have zero or a few labeled text). So when we split the data into train/dev/test, we split by labels rather than text. So, it makes sense that the same piece of text can appear in the train, dev, or test but with different labels.
